bench/wtperf/wtperf.h

```diff
@@ -251,3 +296,14 @@ void	 lprintf(const CONFIG *, int err, uint32_t, const char *, ...)
 __attribute__((format (printf, 4, 5)))
 #endif
 ;
+
+static inline void
```

Comment:
Looks fine to me -- is there something in particular that you're concerned about?